database/seed.js

```diff
@@ -71,4 +73,51 @@ for (restaurantCount = 0; restaurantCount < 100; restaurantCount++) {
     });
 }
 
-// TO DO: Create generated data for Articles table
\ No newline at end of file
+// ARTICLES SEED
+
+let articleData = {
+  articleTitle: '',
+  articleImage: '',
+  articleUrl: '',
+  articleTags: []
+};
+
+let articles = [];
+
+function generateIndividualArticleData() {
+
+  articleData['articleTitle'] = faker.lorem.sentence();
+
+  articleData['articleImage'] = faker.image.nightlife();
+  articleData['articleUrl'] = faker.lorem.slug();
+
+  function getRandomInteger(min, max) {
+    return Math.floor(Math.random() * (max-min + 1)) + min;
+  };
+
+  // STRETCH CONSIDERATION: prevent duplicate faker results
+
+  randomInteger = getRandomInteger(1, 20);
+
+  while (articleData['articleTags'].length < randomInteger) {
+    articleData['articleTags'].push(faker.commerce.productAdjective());
+  }
+
+  return articleData;
+}
+
+for (let articleCount = 0; articleCount < 100; articleCount++) {
+  let articleObject = generateIndividualArticleData();
+
+  let columns = `title, image, url, tags`;
+  let values = '"' + articleData['articleTitle'] + '", ' + '"' + articleData['articleImage'] + '", ' + '"' + articleData['articleUrl'] + '", ' + '"' + articleData['articleTags'] + '"'
```

Comment:
Could we use template strings here instead of concatenation to save some line space?

Reply:
Fantastic suggestion, Ryan! So much cleaner, shorter, and more readable.